Source/WebKit/UIProcess/RemoteLayerTree/RemoteScrollingTree.h

```diff
@@ -58,6 +58,9 @@ class RemoteScrollingTree : public WebCore::ScrollingTree {
     bool scrollingTreeNodeRequestsScroll(WebCore::ScrollingNodeID, const WebCore::RequestedScrollData&) override;
     bool scrollingTreeNodeRequestsKeyboardScroll(WebCore::ScrollingNodeID, const WebCore::RequestedKeyboardScrollData&) override;
 
+    void scrollingTreeNodeWillStartScroll(WebCore::ScrollingNodeID);
```

Comment:
These already exist on the base class:
```
#if PLATFORM(IOS_FAMILY)
    virtual void scrollingTreeNodeWillStartPanGesture(ScrollingNodeID) { }
    virtual void scrollingTreeNodeWillStartScroll(ScrollingNodeID) { }
    virtual void scrollingTreeNodeDidEndScroll(ScrollingNodeID) { }
#endif
```
so don't add new versions here. Move the base class ones outside the #ifdef and fix up the overrides accordingly.